src/compiler/compile/css/Selector.ts

```diff
@@ -250,7 +250,13 @@ function attribute_matches(node: Node, name: string, expected_value: string, ope
 	return false;
 }
 
-function unquote(value: Node) {
+function class_matches(node, name: string) {
```

Comment:
This came back in the merge, but it's not used anymore since #3552.

Reply:
good spot